### ISSUE TITLE
axe-cli: "An error occurred while testing this page"

### ISSUE THREAD

**Issue by DanielTripp** [dequelabs/axe-core-npm#308]
Product:  cli 

Expectation:  To print a list of accessibility violations to stdout.   

Actual:  No list of accessibility violations, or any evidence of a scan, appears on stdout.  Instead, this appears:

<pre><code>
DevTools listening on ws://127.0.0.1:54164/devtools/browser/8e985c0a-b6d8-46ba-aabd-592db8ed88a2
An error occurred while testing this page.
Please report the problem to: https://github.com/dequelabs/axe-core-npm/issues/
</code></pre>

Motivation:  I think that my "Expectation" is a design goal of axe-cli.  Of course, I'm new to this, so I could be wrong.   

<pre><code>
axe-core version: 4.2.0, I think.  (That's what it says in my file C:\Users\dt\axe-cli-july-25-b\axe-core-npm\node_modules\axe-core\axe.js.  First line: "/*! axe v4.2.0".) 
@axe-core/cli: 4.2.2, I think.  (That's what it says in my file C:\Users\dt\axe-cli-july-25-b\axe-core-npm\packages\cli\package-lock.json: "version": "4.2.2".)

- Node version: v14.17.3
- Platform:   Windows  
</code></pre>

I've attached my steps to reproduce as a copy-and-paste of my PowerShell console.
[axe-cli-problem-july-25.txt](https://github.com/dequelabs/axe-core-npm/files/6874257/axe-cli-problem-july-25.txt)

I don't know whether this is a bug or not.  But I sure would benefit from some guidance.   All suggestions welcome.

**Comment by AlexTolok** [comment]
I have same issue with axe cli on Linux inside circleCI
node v14.10.0 (npm v7.20.1)
Running axe-core 4.3.1 in chrome-headless
I have run script against different pages and it always fails (previously it worked fine with same versions and same web pages)  it looks like problem appeared with latest update (as I am installing latest version each time)

**Comment by AlexTolok** [comment]
@DanielTripp if to use older version of chrome it works for me so I think issue appears only with latest chrome (axe cli works fine on chrome v 91.0.4472.164-1 but fails on 92.0.4515.107-1) also it passes if to change browser to firefox 
Try:
`axe   www.deque.com --tags wcag2a --browser firefox --dir ~/axeReports`

**Comment by DanielTripp** [comment]
@AlexTolok Thank you - I tried firefox and it worked.  I looked for older versions of Chrome for download and couldn't find them.  But I'm okay with Firefox for now.

**Comment by dacohenii** [comment]
I think I have the same issue. With the `--show-errors` flag, this is what I get:

```
Error: "session not created: This version of ChromeDriver only supports Chrome version 90\nCurrent browser version is 92.0.4515.107 with binary path /Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 $s SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 90
Current browser version is 92.0.4515.107 with binary path /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
    at Object.throwDecodedError (/Users/dcohen/fk/palantir/palantir/node_modules/selenium-webdriver/lib/error.js:517:15)
    at parseHttpResponse (/Users/dcohen/fk/palantir/palantir/node_modules/selenium-webdriver/lib/http.js:642:13)
    at Executor.execute (/Users/dcohen/fk/palantir/palantir/node_modules/selenium-webdriver/lib/http.js:568:28)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Please report the problem to: https://github.com/dequelabs/axe-core-npm/issues/
```

**Comment by DanielTripp** [comment]
Thank you @dacohenii, I added --show-errors and I also got a message like that:

```
Error: "session not created: This version of ChromeDriver only supports Chrome version 90\nCurrent browser version is 92.0.4515.107 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe"
 $s SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 90
Current browser version is 92.0.4515.107 with binary path C:\Program Files\Google\Chrome\Application\chrome.exe
    at Object.throwDecodedError (C:\Users\dt\AppData\Roaming\npm\node_modules\@axe-core\cli\node_modules\selenium-webdriver\lib\error.js:517:15)
    at parseHttpResponse (C:\Users\dt\AppData\Roaming\npm\node_modules\@axe-core\cli\node_modules\selenium-webdriver\lib\http.js:642:13)
    at Executor.execute (C:\Users\dt\AppData\Roaming\npm\node_modules\@axe-core\cli\node_modules\selenium-webdriver\lib\http.js:568:28)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
Please report the problem to: https://github.com/dequelabs/axe-core-npm/issues/

```

**Comment by Zidious** [comment]
Closing due to inactivity, please re-open if the issue is apparent in 4.4.1
